### DISCUSSION TITLE
How to deploy while setting some configuration (Timeout, Memory)

### DISCUSSION THREAD

**Discussion by Chandrian** [cargo-lambda/cargo-lambda#243] [Q&A] [answered]
Hello,

I see that we can provide the IAM role on the command line, but don't see how to provide the timeout, memory... & other lambda config. 
Is it possible to set those as metadata in the Cargo.toml file? The issue right now is that if I update a function, it set it with the default 30s and create a new version. Could we also set it so that it does not publish a new version?

**Comment by calavera** [comment, marked as answer]
That's a great suggestion. I have opened #244 to discuss possible ways to implement that.
